app/models/user.rb

```diff
@@ -17,6 +17,11 @@ def identifier
     "#{id}:#{username}"
   end
 
+  # Unlock all active locks.
+  def unlock_all!
+    locks.map(&:unlock!)
+  end
```

Comment:
This could be optimized by running just one update statement, but I don't think its a big deal.